core/arch/opnd.h

```diff
@@ -2351,18 +2435,52 @@ DR_API
  * Assumes that \p reg is a DR_REG_ constant.
  * Returns true iff it refers to an xmm (128-bit SSE/SSE2) register
  * or a ymm (256-bit multimedia) register.
+ * \deprecated Prefer reg_is_strictly_xmm().
  */
 bool
 reg_is_xmm(reg_id_t reg);
 
+DR_API
+/**
+ * Assumes that \p reg is a DR_REG_ constant.
+ * Returns true iff it refers to an xmm (128-bit SSE/SSE2) register.
```

Comment:
I know some existing routines have x86-centric docs, but we should try to avoid implicit x86 bias.  Maybe all of these should say "x86" somewhere: "register on x86".  Ditto the others.

Reply:
s/register/x86 register/ for reg_is_xmm, reg_is_strictly_xmm, reg_is_ymm, reg_is_strictly_ymm, reg_is_strictly_zmm.